src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/EnvelopeParserTest.java

```diff
@@ -55,7 +57,7 @@ public void should_throw_an_exception_if_json_is_not_a_valid_envelope() throws E
     }
 
     @Test
-    public void should_throw_an_exception_if_invalid_json_is_passed() throws Exception {
+    public void should_throw_an_exception_if_invalid_json_is_passed() {
```

Comment:
unrelated?

Reply:
yes, it is unrelated to my PR. just cleaned up as the exception is not being thrown